server/src/index.ts

```diff
@@ -37,6 +37,6 @@ router.use('/merchants', merchantsRouter);
 
 app.use('/', router);
 
-const port = process.env.port || 5000;
+const port = process.env.PORT || 8080;
```

Comment:
Whyy :(( My vscode using this port now leh HAHA

Reply:
okay i can change back to 5000 hahaha